performance-tools/performance-cluster/scripts/helpers/create-image.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -x
+#Description: Builds Kyma-Installer image from Kyma sources and pushes it to the repository
+#
+#Expected vars:
+# - KYMA_SOURCES_DIR: directory with Kyma sources to build Kyma-Installer image
+# - KYMA_INSTALLER_IMAGE: Full image name (with tag)
+# - CLOUDSDK_CORE_PROJECT: GCloud Project name, used for KYMA_INSTALLER_IMAGE validation
+#
+#Permissions: In order to run this script you need to use a service account with "Storage Admin" role
+
+set -o errexit
+
+for var in KYMA_SOURCES_DIR KYMA_INSTALLER_IMAGE CLOUDSDK_CORE_PROJECT; do
+    if [ -z "${!var}" ] ; then
+        echo "ERROR: $var is not set"
+        discoverUnsetVar=true
+    fi
+done
+if [ "${discoverUnsetVar}" = true ] ; then
+    exit 1
+fi
+
+echo "--------------------------------------------------------------------------------"
+echo "Building Kyma-Installer image: ${KYMA_INSTALLER_IMAGE}"
+echo "--------------------------------------------------------------------------------"
+echo
+DCONTENT=`ls "${KYMA_SOURCES_DIR}"/tools/kyma-installer/`
```

Comment:
I'm not sure how the output of the folder content helps this script, aside from this it's again a copy of the existing create-image script.